<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Assigned_GameClass</class>
 <widget class="QWidget" name="Assigned_GameClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Uzi_Assigned</string>
  </property>
  <property name="windowIcon">
   <iconset resource="PrePage.qrc">
    <normaloff>:/PrePage/Resource/windowicon.png</normaloff>:/PrePage/Resource/windowicon.png</iconset>
  </property>
  <widget class="QLabel" name="player_img">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>90</width>
     <height>90</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="PrePage.qrc">:/new/game/Resource/player.jpg</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="AI_img">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>0</y>
     <width>90</width>
     <height>90</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="PrePage.qrc">:/new/game/Resource/ai.jpg</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="Thinking_img">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>90</width>
     <height>90</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>570</y>
     <width>303</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="Pass_Btn">
      <property name="text">
       <string>Pass</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>136</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="Regret_Btn">
      <property name="text">
       <string>Regret</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="Info_label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>391</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="PrePage.qrc"/>
 </resources>
 <connections/>
</ui>
